cfme/tests/candu/test_utilization_metrics.py

```diff
@@ -302,3 +308,71 @@ def test_raw_metric_host_disk(metrics_collection, appliance, provider):
         if record.disk_usage_rate_average is not None:
             assert record.disk_usage_rate_average > 0, 'Zero Host Disk IO'
             break
+
+
+def query_metric_rollup_table(appliance, provider, metric, azone_name):
+    metrics_tbl = appliance.db.client['metric_rollups']
+    ems = appliance.db.client['ext_management_systems']
+
+    with appliance.db.client.transaction:
+        provs = (
+            appliance.db.client.session.query(metrics_tbl.id)
+            .join(ems, metrics_tbl.parent_ems_id == ems.id)
+            .filter(metrics_tbl.resource_name == azone_name,
+            ems.name == provider.name)
+        )
+    return appliance.db.client.session.query(metrics_tbl).filter(
+        metrics_tbl.id.in_(provs.subquery()))
+
+
+def generic_test_azone_rollup(appliance, provider, metric, azone_name):
+    query = query_metric_rollup_table(appliance, provider, metric, azone_name)
+
+    for record in query:
+        if getattr(record, metric) is not None:
+            assert getattr(record, metric) > 0, (
```

Comment:
Since we now know that `record` has the attr of `metric`

```suggestion
            assert record.metric > 0, (
```

Reply:
Can we take this one step further and use getattr's default?

Secondly, concern about the TypeError that will result here from the attribute value not being an int?  Easy to wrap it and avoid that just because there's a string.

```
assert int(getattr(record, metric, 0)) > 0
```

Reply:
Done